runtime/doc/options.txt

```diff
@@ -6802,7 +6808,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 			feature}
 	String to put at the start of lines that have been wrapped.  Useful
 	values are "> " or "+++ ": >
-		:set showbreak=>\ 
+		:set showbreak=>\
```

Comment:
Same as above.

Reply:
Thank you for spotting this issue! I removed it unintentionally because I set up vim to automatically remove trailing spaces. I''l send another patch to revert this change.